src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React from 'react';
+import { TodoStatus } from '../../types/TodoStatus';
+
+type Props = {
+  onChangeFilter: (status: TodoStatus) => void;
+  status: TodoStatus;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  onChangeFilter,
+  status,
+}) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        data-cy="FilterLinkAll"
+        href="#/"
+        className={classNames(
+          'filter__link',
+          { selected: status === TodoStatus.ALL },
```

Comment:
Or you can create array from statuses enum and map it here 🙂 
But it's not critica;